astropy/config/tests/test_configs.py

```diff
@@ -16,6 +16,19 @@
 from astropy.utils.exceptions import AstropyDeprecationWarning
 
 
+OLD_CONFIG = {}
+
+
+def setup_module():
```

Comment:
Arguably, all the test in this file should be written such that they do not change the configuration, but this was easier...